lib/git-hooks.rb

```diff
@@ -54,3 +57,17 @@ def real_hook_template_path
     end
   end
 end
+
+# THIS IS A CRAZY TERRIBLE IDEA. This will be added to the original gem though
```

Comment:
Remove comment on code.

If this comment is important, put ir on the commit message plz.